pages/linux/ldconfig.md

```diff
@@ -0,0 +1,15 @@
+# ldconfig
+
+> Configure symlinks and cache for shared library dependencies.
+
+- Update symlinks and rebuild the cache. (Usually run when a new library is installed):
+
+`sudo ldconfig`
+
+- Update symlinks for a given directory:
+
+`sudo ldconfig -n {{path/to/directory}}`
+
+- Print the libraries in the cache and check whether a given library is present:
```

Comment:
Perhaps just including the `ldconfig -p` bit would be ok? Not sure.

Reply:
You would almost always never use `ldconfig -p` without wanting to check for a particular library. This is how the -p flag is commonly used and in the interest of putting useful and practical examples, I have added the grep line.

Reply:
Ahh ok. Looks good to me then!